source/old/osm.rst

```diff
@@ -0,0 +1,33 @@
+Open Street Maps (OSM)
+======================
+.. warning:: This document has been marked for deprecation. The information in the article may be outdated. 
+
+What is OSM?
+------------
+
+OSM is an open source mapping project dedicated to humanitarian mapping. Similar to BTE, they have the goal to map all of the world. While we are working on our Minecraft world, we can contribute to the OSM project and help them to achieve their goal. By editing the OSM database you not only make your own build easier but at the same time you can help identify infrastructure that might be crucial in future humanitarian work. This applies especially if you are working in a developing country where maps are hard to come by.
```

Comment:
Do check out explicit references in reST, cross-referencing across documents is going to be very important